README.rst

```diff
@@ -30,11 +30,18 @@ systems have packages yet, we provide a temporary solution via the
 ``letsencrypt-auto`` wrapper script, which obtains some dependencies
 from your OS and puts others in a python virtual environment::
 
-  user@webserver:~$ git clone https://github.com/letsencrypt/letsencrypt
-  user@webserver:~$ cd letsencrypt
-  user@webserver:~/letsencrypt$ ./letsencrypt-auto --help
+  user@webserver:~$ wget https://dl.eff.org/certbot-auto
+  user@webserver:~$ chmod a+x ./certbot-auto
+  user@webserver:~$ ./certbot-auto --help
 
-Or for full command line help, type::
+.. hint:: The certbot-auto download is protected by HTTPS, which is pretty good, but if you'd like to
+          double check the integrity of the ``certbot-auto`` script, you can use these steps for verification before running it::
+
+            user@server:~$ wget https://dl.eff.org/certbot-auto.sig
+            user@server:~$ gpg2 --recv-key A2CFB51FA275A7286234E7B24D17C995CD9775F2
+            user@server:~$ gpg2 --trusted-key 4D17C995CD9775F2 --verify certbot-auto.sig certbot-auto
```

Comment:
I agree continuing to use OpenSSL signatures is probably the best idea, at least in the short term, however, I bet some of our users don't completely trust the CA system. Other options include

1. Switch le-auto to gpg signatures. Most systems le-auto supports come with gpg by default, however, we'd certainly end up having to do extra work on some systems (e.g. OS X).
2. Do both an openssl and gpg signature during the release process :frowning: 
3. Convert the gpg key to an openssl key for the signature check. I'm not sure how user friendly we can make this or how well we can exploit the PGP web of trust for additional security if we do this.